packages/stable/indent-integration-okta/src/okta-types.ts

```diff
@@ -6,7 +16,11 @@ export interface OktaGroup {
   objectClass: string[]
   type: string
   profile: Profile
-  _links: Links
+  _links: {
+    logo: Link[]
+    users: Link
+    apps: Link
```

Comment:
are these links too?

```suggestion
    users: Link[]
    apps: Link[]
```

Reply:
Nvm — thanks for pointing out https://developer.okta.com/docs/reference/api/groups/#response-example